src/css-files/ResourceCategory.css

```diff
@@ -0,0 +1,24 @@
+#angular-checkbox + label:before{
+    border: 0.2rem solid #000;
```

Comment:
for borders, border-radius, and similar you are allowed to use pixel values since the size of the border won't change if you rescale the website